### PR TITLE
Update IS24 challenge information

### PR DESCRIPTION
**Pull request by ftshijt**

### CODE REVIEW

**Review by simpleoier — APPROVED** [review]
LGTM! Thanks!

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 1]
```diff
@@ -143,52 +143,127 @@ Participation is open to all. Each team can participate in any task. This challe
...
-  * Efficiency of discrete tokens (bitrate): In addition to WER, the efficiency of discrete tokens in the systems will also be evaluated and ranked based on bitrate.
-* Submission
-  * Submission package details:
-    1. The discrete speech units corresponding to the test sets in kaldi format.
-    2. The predicted transcription corresponding to the test sets.
-    3. A technical report in Interspeech2024 paper format (no length limit)
+* Framework: We recommend to use ESPnet for fair comparison. Feel free to let us know your preferrence.
+* Evaluation metrics: 1) Character Error Rates (CERs) on LibriSpeech and ML-SUPERB evaluation sets; 2) the bitrate of the discrete unit.
+  * Character Error Rate (CER): This metric measures the performance of a system in terms of the accuracy of the words recognized or generated compared to a reference. All systems are ranked based on the CERs of the evaluation sets separately: 1) EN: dev-clean / dev-other / test-clean / test-other; 2) ML: test-1h. Note that the ranking of the EN case is based on the aggreggated CER: (total errors of {dev,test}-{clean,other}) / (total length of {dev,test}-{clean,other})
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:148]: "aggreggated" is ambiguous.
Please mention that it is a micro average of all Librispeech test sets.

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 2]
```diff
@@ -143,52 +143,127 @@ Participation is open to all. Each team can participate in any task. This challe
...
-* Submission
-  * Submission package details:
-    1. The discrete speech units corresponding to the test sets in kaldi format.
-    2. The predicted transcription corresponding to the test sets.
-    3. A technical report in Interspeech2024 paper format (no length limit)
+* Framework: We recommend to use ESPnet for fair comparison. Feel free to let us know your preferrence.
+* Evaluation metrics: 1) Character Error Rates (CERs) on LibriSpeech and ML-SUPERB evaluation sets; 2) the bitrate of the discrete unit.
+  * Character Error Rate (CER): This metric measures the performance of a system in terms of the accuracy of the words recognized or generated compared to a reference. All systems are ranked based on the CERs of the evaluation sets separately: 1) EN: dev-clean / dev-other / test-clean / test-other; 2) ML: test-1h. Note that the ranking of the EN case is based on the aggreggated CER: (total errors of {dev,test}-{clean,other}) / (total length of {dev,test}-{clean,other}) 
+* Ranking: The overall ranking is based on the Average Rank, which is the average of all three ranking positions:
+  * R1: aggregated CER on LibriSpeech evaluation sets;
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:150]: ditto

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 3]
```diff
@@ -143,52 +143,127 @@ Participation is open to all. Each team can participate in any task. This challe
...
+  * R2: CER on ML-SUPERB test set;
+  * R3: the bitrate of the overall test sets.
+
+  The overall ranking position is (R1 + R2 + R3) / 3. If more than 1 systems have the same overall ranking position, they are further ranked by the CER of the test-1h.
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:154]: I recommend you add an example for the tie case.

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 4]
```diff
@@ -143,52 +143,127 @@ Participation is open to all. Each team can participate in any task. This challe
...
 * Framework: No framework or model restriction in the TTS-Acoustic+Vocoder challenge, but the organizers have prepared the baseline training scripts (baseline model to be released soon) in [ESPnet](https://github.com/espnet/espnet/tree/tts2/egs2/ljspeech/tts2).
 * Evaluation metrics: Mean cepstral distortion, F0 root mean square error, Bitrate, [UTMOS](https://github.com/sarulab-speech/UTMOS22/tree/master)
-* Ranking:
-  * UTMOS: The primary method for ranking all systems is based on their UTMOS score.
-  * Efficiency of discrete tokens (bitrate): the efficiency of discrete tokens in the systems will also be evaluated and ranked based on bitrate.
+* Ranking: The overall ranking is based on the Average Rank, which is the average of two ranking positions:
+  * R1: UTMOS;
+  * R2: the bitrate of the overall test sets.
+
+  The overall ranking position is (R1 + R2) / 2. If more than 1 systems have the same overall ranking position, they are further ranked by R1.
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:205]: ditto

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 5]
```diff
@@ -143,52 +143,127 @@ Participation is open to all. Each team can participate in any task. This challe
...
 * Framework: No framework or model restriction in TTS-Vocoder challenge, but the organizers have prepared the baseline training scripts (baseline model to be released soon) in [ESPnet](https://github.com/espnet/espnet/tree/tts2/egs2/ljspeech/tts2) and [ParallelWaveGAN](https://github.com/kan-bayashi/ParallelWaveGAN).
 * Evaluation metrics: Mean cepstral distortion, F0 root mean square error, Bitrate, [UTMOS](https://github.com/sarulab-speech/UTMOS22/tree/master)
-* Ranking:
-  * UTMOS: The primary method for ranking all systems is based on their UTMOS score.
-  * Efficiency of discrete tokens (bitrate): the efficiency of discrete tokens in the systems will also be evaluated and ranked based on bitrate.
+* Ranking: The overall ranking is based on the Average Rank, which is the average of two ranking positions:
+  * R1: UTMOS;
+  * R2: the bitrate of the overall test sets.
+
+  The overall ranking position is (R1 + R2) / 2. If more than 1 systems have the same overall ranking position, they are further ranked by R1.
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:242]: ditto

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 6]
```diff
@@ -198,17 +273,35 @@ Participation is open to all. Each team can participate in any task. This challe
...
    * Objective metrics: Mean cepstral distortion, F0 root mean square error, Bitrate for efficiency measure
    * Subjective metrics: Mean Opinion Score (MOS) by organizers
-* Ranking:
-  * MOS: The primary method for ranking all systems is based on their MOS score.
-  * Efficiency of discrete tokens (bitrate): the efficiency of discrete tokens in the systems will also be evaluated and ranked based on bitrate.
+* Ranking: The overall ranking is based on the Average Rank, which is the average of two ranking positions:
+  * R1: MOS;
+  * R2: the bitrate of the overall test sets.
+
+  The overall ranking position is (R1 + R2) / 2. If more than 1 systems have the same overall ranking position, they are further ranked by R1.
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:280]: ditto

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md [hunk 7]
```diff
@@ -229,6 +322,15 @@ Papers for the Interspeech Special Session have to be submitted following the sa
 
 Please use the [submission URL](https://cmt3.research.microsoft.com/INTERSPEECH2024)) to submit papers and select "14.10 Speech Processing Using Discrete Speech Units (Special Session)" as the primary subject areas for your paper submission.
 
+##  Submission pakcage submission
+
+Participants will need to submit their submission package through the [google form](https://forms.gle/XAdQ5WyEVD2tDwYh8), where the leaderboard will be updated accordingly within 48 hours. The final ranking information will be recorded at the time of Interspeech submission deadline (i.e., 2024/03/11 AOE).
```
Comment (sw005320) [inline _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:327]: Can you give me an access to this form?

We should make this part very careful and structured.

**Review by sw005320 — COMMENTED** [review]
Bitrate is very ambiguous.
Also, please define it (provide the equation of how to compute it)

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md (hunk 7 above)
Comment (ftshijt) [reply _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:327]: Sure, send you the access

File: _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md (hunk 1 above)
Comment (ftshijt) [reply _posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md:148]: Fixed in the new PR

**Pull request merged**